lib/poteto/config.rb

```diff
@@ -4,37 +4,43 @@
 
 module Poteto
   class ReviewerConfig
-    attr_reader :exclude, :reviewer
-    attr_accessor :commit_id
+    attr_reader :exclude, :reviewer, :repo
+    attr_accessor :commit_id, :pr_id
 
-    def initialize(reviewer, reviewer_config)
+    def initialize(reviewer, reviewer_config, config)
       @reviewer = reviewer.to_sym
       @exclude = reviewer_config[:exclude] || []
+      @repo = reviewer_config.fetch(:repository)
+      @config = config
+    end
+
+    def access_token
+      @config.access_token
     end
   end
 
   class Config
```

Comment:
Style/Documentation: Missing top-level documentation comment for `class Poteto::Config`.

Reply:
Addressed